atlasopenmagic/metadata.py

```diff
@@ -33,14 +33,6 @@
 import requests
 from requests.exceptions import HTTPError
 
-def warn_with_color(message, category, filename, lineno, file=None, line=None):
-    RED = '\033[91m'
-    RESET = '\033[0m'
-    print(f"{RED}{category.__name__}: {message} ({filename}:{lineno}){RESET}")
```

Comment:
If you prefer to keep this function around to be able to use `warnings.showwarning = atlasopenmagic.metadata.warn_with_color` in user code we can add this back.